container/core.py

```diff
@@ -119,7 +119,7 @@ def cmdrun_build(base_path, project_name, engine_name, var_file=None,
     if engine_obj.service_is_running('conductor'):
         engine_obj.stop_container(conductor_container_id, forcefully=True)
 
-    if not kwargs.get('devel'):
+    if conductor_container_id is None or not kwargs.get('devel'):
```

Comment:
Don't you want this to test for the presence of a Conductor image, not just a container instance?